app/default/routes.py

```diff
@@ -17,7 +17,8 @@
 from flask import request
 from flask import url_for
 from flask_wtf import FlaskForm
-from fsd_utils.authentication.decorators import login_required
+from fsd_utils.authentication.decorators import login_required, login_requested
```

Comment:
Has black run on this and allowed comma separated imports?

Reply:
😮

Reply:
it did! I'm surprised too! let me separate them out for consistency's sake at least 👍🏾